lib/manifest.js

```diff
@@ -290,6 +290,20 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'event/definitions',
+        method: 'event',
+        list: true,
+        useFilename: internals.passthruOn('name')
```

Comment:
Sure!  In the docs I wrote,

> when list is `true` and `place` is a directory without an index file, then this option allows one to use the name of the each file within place to modify its contents. Should be a function with signature `function(filename, value)` that receives the file's filename (without file extension) and its contents at value, returning a new value to be used as arguments for hapi plugin API call.

Imagine `event/listeners`, where calls are made `server.on(criteria, listener)`.  One particular way to make those calls is to put files exporting `{ criteria, listener }` in the folder `event/listeners/`, e.g. `event/listeners/my-event.js`.  If you want to make haute-couture smart enough to use `my-event` as part of `criteria`, then you can write a function `useFilename()` that receives the file's contents (`{ criteria, listener }`) and the filename (`my-event`) and returns the new contents.  In that case we'd want `useFilename` to handle two cases– one, when `criteria` is missing, and two, when `criteria` is an object without a `name` property.  In the former case `internals.passthruOn('criteria')` would be enough, but we'll have to add some additional logic to handle the latter case.

That kinda make sense?  I haven't been awake very long.